src/funfuzz/js/compare_jit.py

```diff
@@ -118,7 +118,7 @@ def compareLevel(jsEngine, flags, infilename, logPrefix, options, showDetailedDi
         # Only used during initial fuzzing. Allowed to have false negatives.
         combos = [combos[0]]
 
-    if flags:
+    if flags and "--no-wasm" not in flags and "--no-wasm-ion" not in flags and "--no-wasm-baseline" not in flags:
```

Comment:
I'm not sure who the caller of this function is. Can it be called with those options, or are we actually the caller?

If it can be called with them, and the order of `flags` isn't important, you could strip these out with `flags = list(set(flags) - {"--no-wasm", "--no-wasm-ion", "--no-wasm-baseline"})`, and then see if anything is left. That would likely be better than silently ignoring `flags`.